sonar-project.properties

```diff
@@ -0,0 +1,14 @@
+sonar.organization=schulcloud-verbund
+sonar.projectKey=hpi-schul-cloud_shd-client
+
+sonar.sources=.
+sonar.tests=.
+
+# Exclude test files, locales and generated code from source scope
+sonar.exclusions=tests/**/*,**/*.unit.ts,**/*.unit.js,src/serverApi/**/*,**/locales/**.ts
```

Comment:
The `tests/**/*` ,`**/*.unit.ts` and `**/*.unit.js` will be automaticly excluded as they add to `test.inclusions`